### PR TITLE
Add test cases for `PersonIsInModulePredicate`

### PR DESCRIPTION
**Pull request by ekweirui**
Currently there exists no test cases for `PersonIsInModulePredicate`.

Thus, this pull request aims to 
* Add test cases for `PersonIsInModulePredicate`

### CODE REVIEW

File: src/main/java/seedu/address/model/module/Module.java [hunk 1]
```diff
@@ -30,45 +30,64 @@ public class Module {
...
+
+    /**
+     * Constructs a {@code Module} with module code, title, tasks and links, but without
+     * persons.
+     */
+    public Module(ModuleCode moduleCode,
+                  ModuleTitle moduleTitle,
                   List<Task> tasks,
                   Set<Link> links) {
         requireAllNonNull(moduleCode, moduleTitle, tasks, links);
```
Comment (cheeheng) [inline src/main/java/seedu/address/model/module/Module.java:59]: Here, perhaps calling `this(moduleCode, moduleTitle, tasks, links, new HashSet<>())` would reduce duplication of code.

File: src/main/java/seedu/address/model/module/Module.java [hunk 2]
```diff
@@ -30,45 +30,64 @@ public class Module {
...
     /**
-     * Adds a {@code Module} with module code, title and links but without
-     * any tasks.
+     * Constructs a {@code Module} with module code, title and links but without
+     * any tasks and persons.
      */
-    public Module(ModuleCode moduleCode, ModuleTitle moduleTitle,
-                  Set<Link> links) {
+    public Module(ModuleCode moduleCode, ModuleTitle moduleTitle, Set<Link> links) {
         requireAllNonNull(moduleCode, moduleTitle, links);
```
Comment (cheeheng) [inline src/main/java/seedu/address/model/module/Module.java:72]: same here

**Review by cheeheng — DISMISSED** [review]

**Review by cheeheng — CHANGES_REQUESTED** [review]
LGTM except for minor nitpicks on code duplication.

File: src/main/java/seedu/address/model/module/Module.java (hunk 1 above)
Comment (teoyuqi) [reply src/main/java/seedu/address/model/module/Module.java:59]: @cheeheng @ekweirui Might it be better to just completely remove the constructor without `persons`, as is in [my PR](https://github.com/AY2223S1-CS2103T-T10-1/tp/pull/90/files#diff-b54a161a8ba7b8350ffbd0a481fc7d3aced24bba452d1b46621499acddea5115)? 

I can't seem to think of any cases whereby we would need the constructor without `persons`. My concern is that leaving it in might introduce bugs, e.g.:
Suppose we already have a set of `Person` to instantiate a `Module` with, but we forget to include it in the  constructor. The constructor without the `persons` argument is called without any bugs. On the other hand, by removing the constructor without the `persons` argument, Java would notice the missing argument and throw an exception.

Comment (teoyuqi) [reply src/main/java/seedu/address/model/module/Module.java:59]: The only concern would be the need to update past code and tests to work using the new constructor. However, I have updated them to use the constructor with `persons` in my PR already.

Comment (ekweirui) [reply src/main/java/seedu/address/model/module/Module.java]: Adopted the `this(...)` approach to reduce duplication of code.

**Review by cheeheng — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/model/module/Module.java [hunk 3]
```diff
@@ -30,45 +31,54 @@ public class Module {
     private final ModuleTitle moduleTitle;
     private final TaskList tasks;
     private final Set<Link> links = new HashSet<>();
```
Comment (teoyuqi) [inline src/main/java/seedu/address/model/module/Module.java:33]:
```suggestion
    private final Set<Link> links;
```
Hmm was there an error in resolving the merge conflict? I believe it was previously agreed that `links`, `persons`  and `tasks` will all be initialised as `null`.

**Review by teoyuqi — APPROVED** [review]
LGTM!

**Review by Tan-Jia-Rong — APPROVED** [review]
LGTM

**Pull request merged**